### PR TITLE
Update use cases in DeveloperGuide.md

### LINKED ISSUE #65: [DG] Update Use Cases

### PR DESCRIPTION
**Pull request by Jo3LW**

### CODE REVIEW

**Review by ElginL — APPROVED** [review]
LGTM 👍

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -308,6 +308,71 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+**Use case: Find a Company**
+
+**MSS**
+
+1. User requests to find a company
+2. AddressBook displays the companies with the same starting names
```
Comment (notle1706) [inline docs/DeveloperGuide.md]: Perhaps you could change AddressBook to JeeqTracker

**Review by notle1706 — COMMENTED** [review]
Couple of minor issues otherwise LGTM

**Review by ElginL — APPROVED** [review]
Nice changes, LGTM

**Review by notle1706 — APPROVED** [review]
LGTM

**Pull request merged**